### PR TITLE
Add unittest for k/k/pkg/kubelet/cm/cgroup_manager_linux.go

### PR DESCRIPTION
**Pull request by zabrox**
#### What type of PR is this?
/kind cleanup

#### What this PR does / why we need it:
This PR adds tests to k/k/pkg/kubelet/cm/cgroup_manager_linux.go.
- Add test for cgroupManagerImpl.Name
- Add test for IsSystemdStyleName
- Append test case for TestParseSystemdToCgroupName

#### Which issue(s) this PR fixes:

Fix part of #109717

#### Does this PR introduce a user-facing change?

```release-note
NONE
```

#### Additional documentation e.g., KEPs (Kubernetes Enhancement Proposals), usage docs, etc.:

```docs

```

### CODE REVIEW

**Review by yangjunmyfm192085 — COMMENTED** [review]
/ok-to-test

**Comment by pacoxu** [comment]
This seems to be fixing only part of https://github.com/kubernetes/kubernetes/issues/109717

Would you change the pr descriptions?
It should not be `Fixes https://github.com/kubernetes/kubernetes/issues/109717`  but something like `fix part of xxx`

**Comment by zabrox** [comment]
@pacoxu 
Thank you for your comment.
I've modified the description.

**Comment by zabrox** [comment]
@yangjunmyfm192085 @pacoxu 
I want this PR to be approved.
I am new to k8s contribution. Is there something I have to do?

**Review by yangjunmyfm192085 — COMMENTED** [review]
/cc @pacoxu
/triage accepted

File: pkg/kubelet/cm/cgroup_manager_linux_test.go [hunk 1]
```diff
@@ -169,3 +173,63 @@ func TestParseSystemdToCgroupName(t *testing.T) {
 		}
 	}
 }
+
+func TestName(t *testing.T) {
```
Comment (mrunalp) [inline pkg/kubelet/cm/cgroup_manager_linux_test.go:177]: TestCgroupName

Comment (mrunalp) [reply pkg/kubelet/cm/cgroup_manager_linux_test.go:177]: Can we make the it more descriptive? TestName is too generic.

**Pull request closed without merging**